app/styles/headers.scss

```diff
@@ -37,6 +37,10 @@
     color: #fff;
 }
 
+a:focus {
+    outline-offset: 1px;
```

Comment:
I see that this outline is used for the contributor names, but it also is affecting the outlines in the top nav (ie. under the user gravatar > My Profile, Settings, etc). The outline looks a little bit too far offset for those, but not terribly so. Not really a problem IMO, just something I had noticed and wanted to bring to your attention.

Reply:
Indeed! That may have been a bit too global. I'll reign it in.